app/Http/Controllers/GalaxyController.php

```diff
@@ -140,17 +147,30 @@ public function getGalaxyArray(int $galaxy, int $system, PlayerService $player,
                     'system' => $system
                 ];
             } else {
+
+                $planet_description = $planetServiceFactory->getPlanetDescription(new Planet\Coordinate($galaxy, $system, $i));
+                $has_colonize_ship = $user_planet->getObjectAmount('colony_ship') > 0 ? "<img src='/img/galaxy/activity.gif' />" . __('galaxy.mission.colonize.no_ship') : '';
+
+                $missions_available = [
+                    [
+                       'missionType' => 0,
+                       'planetMovePossible' => true,
+                       'moveAction' => 'prepareMove',
+                       'title' => 'Relocate'
+                   ]
+                ];
+
+                $missions_available[] = [
+                    'missionType' => 7,
+                    'link'        => $user_planet->getObjectAmount('colony_ship') > 0 ? "/fleet?galaxy={$galaxy}&system={$system}&position={$i}&type=1&mission=7" : '#',
+                    'description' => __('galaxy.mission.colonize.name')."<br>{$planet_description}<br><div>{$has_colonize_ship}</div>"
+                ];
```

Comment:
Currently the error message shows up when the planet has > 0 colony ships. I think this should be reversed: when the planet has exactly 0 the message should show up, when it has more than 0 no message should be displayed.